tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -831,7 +832,7 @@ def show_mo_convert_help():
 
 
 def input_model_is_object(argv):
-    if isinstance(argv['input_model'], str):
+    if isinstance(argv['input_model'], (str, Path)):
```

Comment:
what if I pass numpy array to `convert_model`? Will it treat it as model object? Can we explicitly check model object types which we are supporting?